### ISSUE TITLE
New release to prevent deprecations triggered on PHP 8.1

### ISSUE THREAD

**Issue by christian-kolb** [alcohol/iso3166#75]
I get the following deprecations using version 3.0.0.

```
[2022-07-13T09:21:55.671686+00:00] php.INFO: Deprecated: Return type of League\ISO3166\ISO3166::count() should either be compatible with Countable::count(): int, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice {"exception":"[object] (ErrorException(code: 0): Deprecated: Return type of League\\ISO3166\\ISO3166::count() should either be compatible with Countable::count(): int, or the #[\\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice at /var/www/html/vendor/league/iso3166/src/ISO3166.php:111)"} []
[2022-07-13T09:21:55.672165+00:00] php.INFO: Deprecated: Return type of League\ISO3166\ISO3166::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice {"exception":"[object] (ErrorException(code: 0): Deprecated: Return type of League\\ISO3166\\ISO3166::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice at /var/www/html/vendor/league/iso3166/src/ISO3166.php:123)"} []
```

According to the commit history, this was already [fixed on 19.10.2021](https://github.com/thephpleague/iso3166/commit/5b39a67f74ee9e79085eabb4419ac2d9134bf962) but there hasn't been a release since 05.12.2020. Is it possible to trigger a new bugfix release like 3.0.1 that includes this?

**Comment by techi602** [comment]
Yep, I had to change composer version to dev-main
"league/iso3166": "dev-main",

**Comment by alcohol** [comment]
I am a little bit confused. The fixes you are looking for are available as of `4.0.0`. So why not upgrade to said version?

**Comment by christian-kolb** [comment]
@alcohol Because there is no release with `4.0.0`. I just looked into more details and realized that there is a version that is tagged as `4.0.0` and it's also available in Packagist. I looked into the [releases](https://github.com/thephpleague/iso3166/releases) where it can't be found and didn't think of looking anywhere else 😄
As I'm not the only one stumbling over it, it would be great if it's also listed in the releases.
